apps/desktop/src/lib/components/ProjectSetup.svelte

```diff
@@ -12,15 +12,19 @@
 	import { posthog } from 'posthog-js';
 	import { goto } from '$app/navigation';
 
-	export let remoteBranches: RemoteBranchInfo[];
+	interface Props {
+		remoteBranches: RemoteBranchInfo[];
+	}
+
+	let { remoteBranches }: Props = $props();
 
-	const project = getContext(Project);
+	const project = $state(getContext(Project));
```

Comment:
This $state looks incorrect, you should be able to just remove it?

Reply:
Yeah so this userPrompy thing is a bit of a mess, if I remove that state surrounding the getContext it breaks. There might be some other underlying reactivity bug. 

I've been going through and trying to get rid of all the svelte/legacy imports as this was waiting, will continue with that 👍

Reply:
Removed :ok_hand: